pylon-core/src/main/kotlin/io/github/pylonmc/pylon/core/block/context/BlockBreakContext.kt

```
@@ -21,12 +21,21 @@ interface BlockBreakContext {
     @get:JvmName("normallyDrops")
     val normallyDrops: Boolean
 
+    /**
+     * If true, the block will be set to air after the Pylon data is removed
+     */
+    @Suppress("INAPPLICABLE_JVM_NAME")
```

Uhhh what's this about? That seems important

---

`@JvmName` is not *fully* implemented for interface members, but only for pre-Java 8 targets (something something default interface members something something), but it's fine to just suppress the error if you're targeting Java 8 and later